### DISCUSSION TITLE
Change port during install (Advanced option)

### DISCUSSION THREAD

**Discussion by crazyduck** [tteck/Proxmox#3106] [Ideas]
It would be great, if during install the used port may be changed, probably using advanced settings.

Also raised in a different form here:
https://github.com/tteck/Proxmox/discussions/2972

**Comment by Mellowlynx** [comment]
Not as easy as you think to implement this.
Different apps use different config options, sometimes it's easy, and sometimes it just can't be changed that easy.

You are free to see if it easily can be changed thought the app's config.
But might be better, so start using a reverse proxy, plus sides are that you can use a domain name, add SSL and can just forward to the native port the app uses so updating stays simple.

**Reply by crazyduck** [reply]
agree with most services. Already running a reverse proxy for external reachable services.

In my homenet I would like to use dashy to present all those non 80 port services.
For dashy I modified /opt/dashy/server.js line 48 (in release 3.1.1) from 4000 to 80 and did a rerun of npm run build.


`
> nano /opt/dashy/server.js
line 48: const port = process.env.PORT || (isDocker ? 8080 : 4000);
to
line 48: const port = process.env.PORT || (isDocker ? 8080 : 80);
[strg+x] Y

> npm run build

`
